fusion-builder/fusion-builder-fr_FR.po

```diff
@@ -17,14 +17,17 @@ msgstr ""
 
 #: fusion-builder.php:386
 msgid "Go to the Avada plugin installation page"
-msgstr ""
+msgstr "Aller à la page d'installation du plugin Avada"
```

Comment:
@corend apologies in advance, my French isn't as good as my English or Greek but something doesn't feel right on this line... 
_Aller à la page d'installation du plugin Avada_ feels wrong to me. It feels like it's referring to the Avada plugin which is not the case. And also should it not be plural instead of singular?

Reply:
Singular is correct as you have only one page.
My translation is like "Aller à la page de configuration" --> "Go to the configuration page". This is the same meaning but about the Avada plugin installation page. So for me this is totally correct. That's the same I did for the fr_BE that you approved. :)

Reply:
thanks for the explanation! French is not my primary language so I have to double-check things :laughing: